### PR TITLE
Neka manager dashboard

### LINKED ISSUE #51: Neka-Manager Community Dashboard
This comprises of  Manager Single Community Component, button component

Component Screenshot

update dashboard

### PR DESCRIPTION
**Pull request by istoga**
## What type of PR is this? (check all applicable)

- [x] Feature
- [x] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other (style manager dashboard)
  
  
## Description of what this Pull Request does
- Styled manager's form and manager's dashboard
- Remove add image input from form
- Add a fetch hook to 
- Add a new community
- Delete a community
- Add logout function to managers dashboard

## Related Tickets & Documents
- Closes #51 
- Related Issues: (enter ticket numbers here)
  
## QA Instructions, Screenshots, Recordings

https://github.com/AritDeveloperCircle/findyourpeople/assets/96787210/11366698-806d-454e-92fa-e00123553211

## Link to full video
https://clipchamp.com/watch/elPWgVaHwDl
  
## Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [X] No, and here is why: no testing

### CODE REVIEW

File: src/pages/updateManagerForm.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import Link from "next/link";
```
Comment (msarit) [inline src/pages/updateManagerForm.js:1]: @istoga with these comments in this file, it's safe to say that this PR isn't ready yet eh? 😅

Comment (istoga) [reply src/pages/updateManagerForm.js:1]: Looks like Ibimina is planning some more work off of this PR

Comment (ibimina) [reply src/pages/updateManagerForm.js:1]: @msarit can this pr be merge? I want to work on the unpublished functionality

File: src/pages/managerDashboard.js [hunk 2]
```diff
@@ -42,12 +65,54 @@ const ManagerDashboard = () => {
...
+              <div className="flex flex-col gap-5 p-10  lg:text-left">
+                <h1 className="text-5xl text-blue-800">
+                  Welcome Back, {state?.user?.displayName}
+                </h1>
+                <div className="mt-2">
+                  <Link
+                    href="#"
+                    className="text-slate-700  hover:text-blue-500 cursor-pointer text-2xl"
+                  >
+                    Edit Profile
```
Comment (msarit) [inline src/pages/managerDashboard.js:80]: @istoga Do we have a UI for the "Edit Profile"? 👀

File: src/pages/updateManagerForm.js (hunk 1 above)
Comment (msarit) [reply src/pages/updateManagerForm.js:1]: @ibimina Do you mean Unpublished functionality, or the Approved (to be visible on the landing page) functionality?

I believe we descoped the Publish/Unpublish functionality; have we decided to pick it up again?

Comment (ibimina) [reply src/pages/updateManagerForm.js:1]: I meant the delete/unpublish community

Comment (TaylorDJones11) [reply src/pages/updateManagerForm.js:1]: > 

Hey @ibimina I believe Sheyi wrote a ticket for the delete part. She wants to work on functionality, but I'll assign my unpublish ticket over to you.

File: src/pages/managerDashboard.js (hunk 2 above)
Comment (istoga) [reply src/pages/managerDashboard.js:82]: @msarit We can open another PR and create a modal to pop up when Edit Profile is clicked on. I would love to merge this PR.

**Comment by istoga** [comment]
@msarit can we merge this?

**Comment by msarit** [comment]
@istoga @ibimina @TaylorDJones11 @Sheyitofunmi
I can merge this PR in as-is, seeing that we are down to the wire.

However there are a few issues I encountered:
- I can still access the Dashboard without being signed in
- The Logout button in the dashboard does not work (also why do we not have the Nav Bar in the Dashboard view?)
- The "Add new community" button is not working

Let me know if you want to fix these observations, or I should merge as-is. 🙏🏾

**Comment by ibimina** [comment]
> @istoga @ibimina @TaylorDJones11 @Sheyitofunmi I can merge this PR in as-is, seeing that we are down to the wire.
> 
> However there are a few issues I encountered:
> 
> * I can still access the Dashboard without being signed in
> * The Logout button in the dashboard does not work (also why do we not have the Nav Bar in the Dashboard view?)
> * The "Add new community" button is not working
> 
> Let me know if you want to fix these observations, or I should merge as-is. 🙏🏾
> 
> PS I resolved this PRs conflicts with `main`

@istoga said you suggested that the we remove the search bar since it not working properly, the best to implement the search will be to redirect to a search page when a search is made

**Review by ibimina — APPROVED** [review]

**Review by msarit — APPROVED** [review]

**Pull request merged**